src/handlers/jira.ts

```diff
@@ -108,9 +106,88 @@ export class JiraHandler extends BaseHandler {
+            "fields": {
+                "summary": "Sonatype IQ Evaluation Results",
+                "issuetype": {
+                    "name": target.issueType
+                },
+                "project": {
+                    "key": target.projectKey
+                },
+                "description": {
+                    "type": "doc",
+                    "version": 1,
+                    "content": [
+                        {
+                            "type": "paragraph",
+                            "content": [
+                                {
+                                    "type": "text",
+                                    "text": "Sonatype Scan Result for "
+                                },
+                                {
+                                    "type": "text",
+                                    "text": payload.applicationEvaluation.application.name,
+                                    "marks": [
+                                        {
+                                            "type": "strong"
+                                        },
+                                        {
+                                            "type": "em"
+                                        }
+                                    ]
+                                },
+                                {
+                                    "type": "text",
+                                    "text": `\n Application Evaluation Report:`,
+                                    "marks": [
+                                        {
+                                            "type": "strong"
+                                        }
+                                    ]
+                                },
+                                {
+                                    "type": "text",
+                                    "text": `\n\t- Affected Components:\t ${payload.applicationEvaluation.affectedComponentCount} \n\t- Critical Issues:\t ${payload.applicationEvaluation.criticalComponentCount} \n\t- Severe Issues:\t ${payload.applicationEvaluation.severeComponentCount} \n\t- Moderate Issues:\t ${payload.applicationEvaluation.moderateComponentCount} \n Evaluation Date: ${payload.applicationEvaluation.evaluationDate} \nStage: ${payload.applicationEvaluation.stage} \nOutcome: ${payload.applicationEvaluation.outcome}`,
+                                },
+                                {
+                                    "type": "text",
+                                    "text": "\nView Evaluation in Sonatype IQ",
+                                    "marks": [
+                                        {
+                                            "type": "link",
+                                            "attrs": {
+                                                "href": getIqUrlForApplicationEvaluation(payload),
+                                                "title": "IQ URL"
```

Comment:
Should this not be more on brand? Sonatype Lifecycle?

Reply:
Done!